generator/README.md

```diff
@@ -41,14 +41,29 @@ Put the extracted mibs in a location NetSNMP can read them from. `$HOME/.snmp/mi
 
 https://github.com/librenms/librenms/tree/master/mibs can also be a good source of MIBs.
 
-
+## Generator file format
```

Comment:
This needs work generally, I'd leave it to another PR. For example all the auth stuff needs to move over here.